builder/templates/builder/updates.js

```diff
@@ -0,0 +1,10 @@
+function scrollToBottomOf(element) {
+    const rect = element.getBoundingClientRect();
+    window.scroll(0, window.scrollY + rect.bottom - window.innerHeight);
+}
+
+const buttons = document.querySelectorAll("button.scroll-button");
+buttons.forEach((b) => b.onclick = (event) => {
```

Comment:
Instead of `onclick`, `addEventListener` is the recommended newer API.

https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/addEventListener